src/media/js/addon/actions/addon.js

```diff
@@ -71,3 +80,28 @@ export function fetchVersions(addonSlug) {
       });
   };
 }
+
+
+export function install(addonSlug, manifestUrl) {
+  /*
+    Install add-on.
+  */
+  return (dispatch, getState) => {
+    const apiArgs = getState().apiArgs || {};
+    manifestUrl = Url(manifestUrl).q(apiArgs);
```

Comment:
Would be super cool if we could combine this pattern into a single function, we do it all over the place.